priv/json-schemas/schemas/list/recipients.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "oneOf": [
+      {
+        "$ref": "file://schemas/base/address.json"
+      },
+      {
+        "$ref": "file://schemas/object/recipient.json"
+      }
+    ]
+  },
+  "maxItems": 255
```

Comment:
@samuelmanzanera What are you thoughts on this maxItems=255? In the serialization we have no such limitation